loop_rate:      10.0  # The frequency in Hz at which to run the control loop.
queue_size:     1     # The size of the message queue used for publishing and subscribing to topics.
yaw_tolerance:  0.25  # The angle in radian which the yaw of the CPS can vary around the set point.
turn_timeout:   5.0   # The time in seconds that the turning is allowed to take.
turning:        false # Whether the CPS should turn its front into movement direction or not.
visualize:      true  # Whether to publish the waypoints on a topic for visualization.
